src/graph/GraphViewServer.ts

```diff
@@ -135,50 +136,51 @@ export class GraphViewServer extends EventEmitter {
     const start = Date.now();
 
     try {
-      await this._actionHandler.callWithTelemetry("cosmosDB.gremlinQuery", async (telemetryProperties, measurements) => {
-        this._pageState = {
+      const graphViewServier: GraphViewServer = this;
```

Comment:
nit: a common pattern for this is:
  let me = this;
or
  let thisServer = this;

Reply:
Also, you misspelled server :-)